docs/web/docker.md

```diff
@@ -110,3 +123,19 @@ previously created password files for secrets.
 ### Running your app
 Run `docker-compose -f web/docker/services/<service-yml-file> up -d` and
 Compose starts and runs your entire app.
+
+## Kubernetes
+[![Kubernetes](../images/kubernetes.png)](https://kubernetes.io/)
+
+CodeChecker supports to configure liveness, readiness and startup probes for
+Containers.
```

Comment:
```suggestion
containers.
```
I think a link to https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/ would be nice somewhere here.